pkg/kubelet/cm/helpers.go

```diff
@@ -21,7 +21,7 @@ import (
 	evictionapi "k8s.io/kubernetes/pkg/kubelet/eviction/api"
 )
 
-// hardEvictionReservation returns a resourcelist that includes reservation of resources based on hard eviction thresholds.
+// hardEvictionReservation returns a resource list that includes reservation of resources based on hard eviction thresholds.
```

Comment:
I think `ResourceList` would be more appropriate here.

Reply:
Although this function returns a 'v1.ResoureList' object, this comment seems to emphasize the data structure of the list, 
So I think the lowercase 'resourcelist' is not suitable here......